R/package.R

```diff
@@ -1,10 +1,8 @@
-
-#' Advice on R Package Building
+#' goodpractice: Advice on R Package Building
 #'
 #' Give advice about good practices when building R packages. Advice includes
 #' functions and syntax to avoid, package structure, code complexity, code
 #' formatting, etc.
-#'
-#' @docType package
-#' @name goodpractice
-NULL
+#' @name goodpractice-package
+#' @aliases NULL
+"_PACKAGE"
```

Comment:
The `NULL` alias avoids the conflicts between the `goodpractice` function page and the package name page. The "_PACKAGE" takes care of adding `@docType` parameter. This is the more updated version of documenting packages, AFAIK.